tests/wasi/testdata/args.go

```diff
@@ -0,0 +1,14 @@
+package main
+
+import (
+	"fmt"
+	"os"
+)
+
+func main() {
+}
+
+//export PrintArgs
+func PrintArgs() {
```

Comment:
please remove this and just call it from main as the comments clarifying this are completely distracting. MAY is not an issue except if we touch something. avoid that and just inline the printf in main.

as mentioned before, to call the function directly is perfectly in our ability to control and allow, and calling that directly should be in a unit test as "tests" won't necessarily ever be run by someone on change